resources/default_pipeline_environment.yml

```diff
@@ -493,7 +493,7 @@ steps:
       dockerName: 'bundler'
       dockerWorkspace: ''
   slackSendNotification:
-    color: "${buildStatus == 'SUCCESS'?'#008000':'#E60000'}"
+    color: "${['SUCCESS': '#8cc04f', 'FAILURE': '#d54c53', 'ABORTED': '#949393', 'UNSTABLE': '#f6b44b'].get(buildStatus)}"
```

Comment:
How is the behavior in case the value in not in the map?

Reply:
i use now the unknown color as default